tests/test_radar_dataset.py

```diff
@@ -58,3 +58,12 @@ def test_radar_dataset(tmp_path):
     with pytest.raises(ValueError) as excinfo:
         RadarDataset(tmp_path, features, target, min_neighbours=10**8)
     assert "No graphs" in str(excinfo.value)
+
+    # Test with a explicit string as argument for folder.
+    dataset = RadarDataset(
```

Comment:
Maybe we should use fixtures for this as described here https://github.com/point-cloud-radar/bird-cloud-gnn/issues/51?

Reply:
This is testing `RadarDataset` functionalities (i.e., it can be created by passing a string for the folder name), so it shouldn't be a fixture. Unless you mean something else.